brainvisa/toolboxes/tools/processes/documentation/generateDocumentation.py

```diff
@@ -103,28 +104,28 @@ def generateHTMLDocumentation(processInfoOrId, translators, context, ontology):
         if not os.path.isdir(p):
             os.makedirs(p)
         # Main page
-        f = open(htmlFileName, 'w')
-        print('<html>', file=f)
-        print('<head>', file=f)
-        print('<title>' + tr.translate(processInfo.name) + '</title>', file=f)
+        f = io.open(htmlFileName, 'w', encoding='utf-8')
+        f.write(six.text_type("""
```

Comment:
I would recommend to use Unicode strings directly (`u'string'`) rather than using native strings and converting them with `six.text_type`: it is more straightforward, and there could be corner-cases that `six.text_type` does not handle well.

Reply:
(same thing for other calls to `six.text_type` below)